cmd/login.go

```diff
@@ -56,9 +56,11 @@ func init() {
 	viper.AddConfigPath("/etc/mender-cli/")
 	viper.AddConfigPath("$HOME/")
 	viper.AddConfigPath(".")
-	err := viper.ReadInConfig()
-	if err != nil {
-		log.Info("No configuration file found")
+	if err := viper.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
```

Comment:
Suggestion:
Import the `errors` package, and do:
```
if errors.Is(err, viper.ConfigFileNotFoundError) {
...
```

Reply:
`type viper.ConfigFileNotFoundError is not an expression`. ConfigFileNotFoundError is basically is not of type error.

Reply:
`viper.ConfigFileNotFoundError` is a struct. So it needs the:
```
errors.Is(err, viper.ConfigFileNotFoundError{})
```

Reply:
You are right! Thx!

Reply:
First errors.Is internally compares if they are comparable (check if type compares). If true, it checks if `err == viper.ConfigFileNotFoundError{}`. Problem is that `err` contains data and the empty struct for `ConfigFileNotFoundError` doesn't this check won't succeed - ever.

Reply:
Ahh, I see. Alright, let's leave it as it is then